app/lib/rocketchat.js

```diff
@@ -90,7 +90,22 @@ const RocketChat = {
 	},
 	async getServerInfo(server) {
 		try {
-			const result = await fetch(`${ server }/api/info`).then(response => response.json());
+			let result = await fetch(`${ server }/api/info`)
+				.then(response => response.text());
+			try {
+				result = JSON.parse(result);
+				if (!{}.hasOwnProperty.call(result, 'success')) {
+					return {
+						success: false,
+						message: 'Not_is_rc'
+					};
+				}
+			} catch (e) { // if returns html on body
```

Comment:
It could fail because of other reasons, so I don't think this comment is necessary.